app.lua

```diff
@@ -126,8 +134,18 @@ function app:handle_404()
 end
 
 function app:handle_error(err, trace)
+    -- self.current_user is not available here.
+    local current_user = package.loaded.Users:find(self.session.username)
+    if current_user then
+        user_params = current_user:rollbar_params()
+    else
+        user_params = {}
+    end
     print(err)
     print(trace)
+    rollbar.set_person(user_params)
+    rollbar.set_custom_trace(err .. "\n\n" .. trace)
```

Comment:
This looks a little funny, but both variables contain different parts of the trace. And normalize_error method helps, but in the case of DB errors, doesn't really grab the best line, so I want to make sure everything is logged.